packages/remix/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "@supabase/auth-helpers-remix",
+  "version": "0.4.0",
```

Comment:
You might want to reset this version number to 0.0.0 and make the changeset a minor release which will bump it to 0.1.0 in the GitHub action process.

Reply:
Good call!